_data/sites.json

```diff
@@ -3636,6 +3636,17 @@
         ]
     },
 
+    {
+        "name": "Fontawesome",
+        "url": "https://fontawesome.com/privacy",
+        "difficulty": "hard",
+        "email": "privacy@fontawesome.com",
+        "notes": "Canceling a PRO acconut deletes the account. Free accounts must send an e-mail.",
```

Comment:
```suggestion
        "notes": "Cancelling a PRO account already deletes it. Free accounts must be deleted by sending an e-mail.",
```

Reply:
I'd keep the first sentence, while taking the second's change. Whatever you want for Cancel(l)ing.